.github/workflows/release.yaml

```diff
@@ -149,4 +149,33 @@ jobs:
           name: v${{ env.VERSION }}
           draft: true
           prerelease: ${{ startsWith(env.VERSION, '0') }}
-          files: artifacts/*
\ No newline at end of file
+          files: artifacts/*
+
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    needs: [ prepare ]
+    if: ${{ needs.prepare.outputs.found == 'false' }}
```

Comment:
When we bump the registry version without bumping the CLI, we don't want to publish to crates.io, which is what the found output represents. This condition makes it so we can't release a new registry version without a CLI bump.

Reply:
I see, so should that check be removed?

Reply:
Yes.